core/server/src/main/webapp/general.jsp

```diff
@@ -114,6 +114,38 @@
       </div>
     </div>
   </div>
+  <!--  show inconsistent path  -->
+  <% if ((Integer) request.getAttribute("inconsistentPaths") != 0) { %>
+  <div class="row-fluid">
+    <div class="accordion span14" id="accordion4">
+      <div class="accordion-group">
+        <div class="accordion-heading">
+          <a class="accordion-toggle" data-toggle="collapse" data-parent="#accordion4" href="#data4">
+            <h4>Inconsistent Files</h4>
```

Comment:
I fully agree, with going for option 1 in this PR. For the more advanced function of refresh, this ticket is already open: https://alluxio.atlassian.net/browse/ALLUXIO-2411.